### DISCUSSION TITLE
Is there any example code for attacking the speech recognition models (DeepSpeech/Lingvo)?

### DISCUSSION THREAD

**Discussion by xinghua-qu** [Trusted-AI/adversarial-robustness-toolbox#1234] [Q&A] [answered]
I see the estimator provides some speech recognition models (e.g., DeepSpeech and Lingvo). May I ask is there any example code for attacking those models?

**Comment by beat-buesser** [comment, marked as answer]
Hi @QuXinghuaNTU Thank you very much for your interest in ART!

Currently we have multiple notebook related to ART and audio/ASR:
- [adversarial_audio_examples.ipynb](https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/main/notebooks/adversarial_audio_examples.ipynb)
- [asr_deepspeech_examples.ipynb](https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/main/notebooks/asr_deepspeech_examples.ipynb)

Btw, ART 1.7 also provides an estimator for [Espresso](https://github.com/freewym/espresso).

**Reply by xinghua-qu** [reply]
Thanks for the quick reply. That greatly helps:)
